Tests/MongoSwiftTests/CommandMonitoringTests.swift

```diff
@@ -10,6 +10,11 @@ final class CommandMonitoringTests: MongoSwiftTestCase {
     }
 
     func testCommandMonitoring() throws {
+        if MongoSwiftTestCase.topologyType == .sharded {
```

Comment:
i think `guard` is more idiomatic in these cases where we return early inside an `if`, but since we will hopefully delete these skips soon anyway i'm fine with leaving as is.